module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -783,10 +819,13 @@ public function getHolding($bibId, array $patron = null, array $options = [])
             foreach (
                 $this->getPagedResults(
                     'items',
-                    '/item-storage/items',
+                    '/inventory/items-by-holdings-id',
                     $query
                 ) as $item
             ) {
+                if ($item->discoverySuppress) {
+                    continue;
+                }
```

Comment:
`/inventory/items-by-holdings-id` is an optimized call to retrieve both directly linked and bound-with items at once.  They are distinguished in the result set by the isBoundWith attribute.  The downside is this API does not support passing the discoverySuppress parameter so that filter is done to the results intead.